homeassistant/components/overkiz/entity.py

```diff
@@ -100,6 +125,12 @@ def __init__(
     ) -> None:
         """Initialize the device."""
         super().__init__(device_url, coordinator)
+
+        if self.is_sub_device and self.is_sub_device_default_naming:
+            # In case of sub device  with description using the default naming, use the provided label and append the name of the type of entity
+            description = copy.copy(description)
+            description.name = f"{self.device.label} {description.name}"
```

Comment:
Why not just set `self._attr_name` here using the description name as suffix? Modifying the description here isn't more readable than setting `self._attr_name`.

Reply:
Because I thought it was I was asked to do here: https://github.com/home-assistant/core/pull/79229#issuecomment-1262990264
Revert it back to simpler _attr_name setting